tests/test_counter.py

```diff
@@ -0,0 +1,21 @@
+"""This module test the counter.py module regarding if statements"""
+from castanet import counter
+from castanet import generate_trees as generator
+
+def test_match_if_statements_1():
+    """Uses match_if_statements to identify all the if statements in the test_files directory"""
+    directory = "./test_files"
+    file_list = generator.find_python_files(directory)
+    string_file_list = generator.read_files(directory, file_list)
+    tree_dict = generator.generate_cast(string_file_list)
+    if_dictionary = counter.match_if_statements(tree_dict)
+    assert if_dictionary == {'blackjack.py': 5}
+
+def test_match_if_statements_2():
+    """Uses match_if_statements to identify all the if-statements in the hello directory"""
+    directory = "./hello"
+    file_list = generator.find_python_files(directory)
+    string_file_list = generator.read_files(directory, file_list)
+    tree_dict = generator.generate_cast(string_file_list)
+    if_dictionary = counter.match_if_statements(tree_dict)
+    assert if_dictionary == {'say_hello.py': 1, '__init__.py': 0}
```

Comment:
These test cases are going to fail once my `setup` branch is merged to `devlop` as I had to add UTF-8 encoding to the file names. As a result, the file names are much more complex now. I would suggest only checking length of dictionaries for this round of test cases until we figure this out.